samples/web/content/apprtc/appwindow.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
```

Comment:
I did this work in another branch:
https://github.com/chuckhays/webrtc/commit/e1be7f0e016453a8585319eef7e601d3587595cb
Do you want me to merge it to this branch, or do it as a separate PR after this one is completed?